### PR TITLE
fix: Stop making build successful if tests fail

### PR DESCRIPTION
**Pull request by devchan4188**
Closes: dequelabs/axe-android-app#125

## Requester Review Items

- [x] Ensure the Pull Request is into develop.
- [x] Test coverage has not gone down.
- [x] Documentation updated or not needed. (Wiki, Issues, etc)
- [x] Angular Type leads to proper Semantic Version.

## Dev Team Review Items

To be filled out by @dequelabs/mobile.

- [x] Ensure the Requester did not lie. Chastise them politely if they did.
- [x] Code Quality review.
- [x] Changes to Axe*.java classes are minimal, appropriate, and versioned properly.
  - [x] Serialized Axe objects have not changed.
  - [x] Public API has not changed.

## Code Owner Review Items

To be filled out by @dequelabs/mobileadmin.

- [ ] Any Rules package changes lead to proper Semantic Version.
- [ ] Security Review.

## Merger Review Items

- [ ] Ensure commit message matches title before squashing.

### CODE REVIEW

File: .circleci/env.sh [hunk 1]
```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/env bash
 
-export git=$(sh /etc/profile; which git)
-
-cd "${0%/*}"
-
-export TAB=$'\t'
-export NEW=$'\n'
+#export git=$(sh /etc/profile; which git)
```
Comment (cyono) [inline .circleci/env.sh:3]: Commented code

File: .circleci/env.sh [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 ROOT_PATH=${CIRCLE_CI_PATH}/../
 ARTIFACTS_PATH=${CIRCLE_CI_PATH}/artifacts
 
 cd ${ROOT_PATH}
 
-function debug() {
-  >&2 echo "DEBUG:$1"
-}
\ No newline at end of file
+#function debug() {
```
Comment (cyono) [inline .circleci/env.sh:16]: As above

**Comment by cyono** [comment]
This PR began codeowner file work, closing this so we can open a more accurate and relevant PR.

**Comment by devchan4188** [comment]
> This PR began codeowner file work, closing this so we can open a more accurate and relevant PR.

@cyono I have removed the CODEOWNER file changes from this PR, will create a separate PR for the CODEOWNER changes.

File: .circleci/env.sh (hunk 1 above)
Comment (cyono) [reply .circleci/env.sh:3]: Still here

File: .circleci/env.sh (hunk 2 above)
Comment (cyono) [reply .circleci/env.sh:16]: This, too

**Pull request merged**